### PR TITLE
Add blurb about root names

### PR DESCRIPTION
**Pull request by webbushka**
## Summary by CodeRabbit

- **Documentation**
	- Refined the description of the name service hierarchy.
	- Clarified the normalization process for name records.
	- Introduced guidelines for creating root names.
	- Corrected spelling and punctuation errors.

### CODE REVIEW

**Review by SpicyLemon — DISMISSED** [review]

**Review by Taztingo — DISMISSED** [review]

**Comment by webbushka** [comment]
I don't have auth to merge these after they are approved if someone is willing to merge that would be appreciated.

@scirner22 @Taztingo @SpicyLemon

**Comment by SpicyLemon** [comment]
> I don't have auth to merge these after they are approved if someone is willing to merge that would be appreciated.
> 
> @scirner22 @Taztingo @SpicyLemon

I was waiting for the coderabbit comments to either be applied or resolved/dismissed.

**Comment by webbushka** [comment]
> > I don't have auth to merge these after they are approved if someone is willing to merge that would be appreciated.
> > @scirner22 @Taztingo @SpicyLemon
> 
> I'm waiting for the coderabbit comments to either be applied or resolved/dismissed.

All completed

**Review by SpicyLemon — APPROVED** [review]

**Pull request merged**
